commit 17a3e5bffb7110d46de1bf42b64b90713ff5ea50
Author: Chris Mayo <aklhfex@gmail.com>
Date:   Sun Feb 1 12:03:46 2015 -0500

    statd: Fix test for foreground mode
    
    daemon_init parameter has the opposite sense
    to code removed in commit 7addf9d
    
    Signed-off-by: Chris Mayo <aklhfex@gmail.com>
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/statd/statd.c b/utils/statd/statd.c
index 60ce6d1..2b7a167 100644
--- utils/statd/statd.c
+++ utils/statd/statd.c
@@ -393,7 +393,7 @@ int main (int argc, char **argv)
 		simulator (--argc, ++argv);	/* simulator() does exit() */
 #endif
 
-	daemon_init(!(run_mode & MODE_NODAEMON));
+	daemon_init((run_mode & MODE_NODAEMON));
 
 	if (run_mode & MODE_LOG_STDERR) {
 		xlog_syslog(0);
